AttributeError: 'dict' object has no attribute 'strip'

The following needs to be resolved:

```bash
[ERROR] AttributeError: 'dict' object has no attribute 'strip'
Traceback (most recent call last):
  File "/var/task/lambda.py", line 160, in lambda_handler
    document_delete_range    = json.loads(properties.get('DocumentDeleteRange', '
{}
').strip())
```

--- a/lambda.py
+++ b/lambda.py
@@ -157,7 +157,7 @@ def lambda_handler(event, context, physicalResourceId=None, noEcho=False):
     )).strip()
     mappings                 = json.loads(properties.get('Mappings', '{}').strip())
     initialize_dashboard     = bool(strtobool(properties.get('InitalizeDashboard', 'False').strip().capitalize()))
-    document_delete_range    = json.loads(properties.get('DocumentDeleteRange', '{}').strip())
+    document_delete_range    = json.loads(properties.get('DocumentDeleteRange', '{}'))
     executions               = []
 
     #

